Tests/LinuxMain.swift

```diff
@@ -18,9 +18,15 @@ import XCTest
 
 @testable import KituraSessionTests
 
-
+#if swift(>=4.1)
```

Comment:
This might be better (uncompiled):

```swift
var testcases = [testCase(TestSession.allTests), testCase(TestTypeSafeSession.allTests]
#if swift(>=4.1)
testcases.append(TestCodableSession.allTests)
#endif

XCTMain(testcases)
```